config/sync/ultimate_cron.job.node_cron.yml

```diff
@@ -0,0 +1,17 @@
+uuid: 81c14dfa-c0af-4160-a905-776f02e0832c
```

Comment:
These config ids may conflict with https://github.com/SU-SWS/stanford_profile/pull/212 but as long as they get in before we punch a release it should be fine.